### ISSUE TITLE
Some question about training the yolo-nas-pose model

### ISSUE THREAD

**Issue by ZYX-MLer** [Deci-AI/super-gradients#1807]
### 💡 Your Question

I have five questions about the yolo-nas-pose model: 

1、I noticed that the default parameter file 'coco2017_yolo_nas_pose_s' gives results based on 8 Nvidia RTX 3090 GPUs. If I only have one RTX 3090, how should I adjust these parameters? 

2、I observed that the 'batch_size' in the file is set to 48 and num_gpus is 8. Does this mean that each iteration uses samples equivalent to 48 times 8? Hence, I need to decrease the learning rate to 1/8 if i use only one 3090? 

3、If I reduce the 'initial_lr' by a factor of 8, should I increase the 'cosine_final_lr_ratio' by a factor of 8?

4、I am training a new YOLO-NAS-Pose model from scratch, and I noticed that the default parameter for epochs is set to 1000, with early stop incorporated. What is the typical number of epochs to achieve reasonably good results?

5、why the padding_mode of KeypointsPadIfNeeded in the training dataset is 'center', while in the val dataset is 'bottom_right'.?

Thank you.

**Comment by BloodAxe** [comment]
> 1、I noticed that the default parameter file 'coco2017_yolo_nas_pose_s' gives results based on 8 Nvidia RTX 3090 GPUs. If I only have one RTX 3090, how should I adjust these parameters?

You may want to lower learning rate accordingly

> 2、I observed that the 'batch_size' in the file is set to 48 and num_gpus is 8. Does this mean that each iteration uses samples equivalent to 48 times 8? Hence, I need to decrease the learning rate to 1/8 if i use only one 3090?

Batch size is defined per 1 GPU. 

> 3、If I reduce the 'initial_lr' by a factor of 8, should I increase the 'cosine_final_lr_ratio' by a factor of 8?

No, since it's a ratio and not an absolute value

> 4、I am training a new YOLO-NAS-Pose model from scratch, and I noticed that the default parameter for epochs is set to 1000, with early stop incorporated. What is the typical number of epochs to achieve reasonably good results?

You should observe the AP plots, for 8 GPU it was something around 200-300 when it started reaching platoe.

> 5、why the padding_mode of KeypointsPadIfNeeded in the training dataset is 'center', while in the val dataset is 'bottom_right'.?

There is no specific reason why it done like that.
